addons/delivery/models/delivery_carrier.py

```diff
@@ -108,6 +108,12 @@ def _match_address(self, partner):
             return False
         return True
 
+    def get_carrier_name_amazon(self):
+        '''Amazon changed its policy to accept carrier names, and it is a bit more restrictive than expected.
+        This method will return the carrier name as a string.
```

Comment:
```suggestion
        ''' Hook to send a compliant carrier name when needed
        
        This function needs to be overridden in the delivery carriers.
```

This is a commit message, not what this function does. This function is a hook that need to be overridden in other modules. We don't want to speak of Amazon in Delivery.